apps/storybook/src/VisCanvasInteraction.stories.tsx

```diff
@@ -1,26 +1,25 @@
 import { PanZoomMesh, TooltipMesh, VisCanvas } from '@h5web/lib';
 import type { TooltipMeshProps } from '@h5web/lib';
+import type { PanZoomProps } from '@h5web/lib/src/vis/shared/PanZoomMesh';
 import { formatTooltipVal } from '@h5web/shared';
 import type { Meta, Story } from '@storybook/react';
 
 import VisCanvasStoriesConfig from './VisCanvas.stories';
 
-interface TemplateProps {
-  panZoom?: boolean;
-  panKey?: 'Alt' | 'Control' | 'Shift';
+interface TemplateProps extends PanZoomProps {
   tooltipValue?: string;
   guides?: TooltipMeshProps['guides'];
 }
 
 const Template: Story<TemplateProps> = (args) => {
-  const { panZoom = false, tooltipValue, guides, panKey } = args;
+  const { tooltipValue, guides, ...panZoomProps } = args;
 
   return (
     <VisCanvas
       abscissaConfig={{ visDomain: [-10, 0], showGrid: true }}
       ordinateConfig={{ visDomain: [50, 100], showGrid: true }}
     >
-      {panZoom && <PanZoomMesh panKey={panKey} />}
+      <PanZoomMesh {...panZoomProps} />
```

Comment:
`panZoom` is no longer necessary in the story as we can equivalently disable panning and zooming though the props `pan` and `zoom`